### PR TITLE
Run pytest in conda recipe

### LINKED ISSUE #222: Run pytest in conda recipe

### PR DESCRIPTION
**Pull request by 1e-to**
Closes #222

### CODE REVIEW

File: conda-recipe/meta.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ requirements:
 
 test:
   requires:
-    - scipy  # [linux]
+    - pytest
```
Comment (PokhodenkoSA) [inline conda-recipe/meta.yaml:36]: Why scipy is removed?

Comment (1e-to) [reply conda-recipe/meta.yaml:36]: pytest can runs without it
Return back?

Comment (PokhodenkoSA) [reply conda-recipe/meta.yaml:36]: Did you run it with dpnp?

Comment (PokhodenkoSA) [reply conda-recipe/meta.yaml:36]: @1e-to update to `main` please.

**Comment by PokhodenkoSA** [comment]
Command to run tests in dpCtl:
https://github.com/IntelPython/dpctl/blob/77bc086561e87770eef5237d58b46c1564f290c8/conda-recipe/run_test.sh#L9

**Review by PokhodenkoSA — APPROVED** [review]
LGTM

**Pull request merged**